app/src/js/reducers/operations.js

```diff
@@ -60,9 +59,9 @@ export default createReducer(initialState, {
     delete state.list.params[action.paramKey];
   },
   [SEARCH_OPERATIONS]: (state, action) => {
-    state.list.internal.infix = action.infix;
+    state.list.params.infix = action.infix;
   },
   [CLEAR_OPERATIONS_SEARCH]: (state) => {
-    delete state.list.internal.infix;
+    state.list.params.infix = null;
```

Comment:
So `internal` was redundant to `params`?

Reply:
internal is not used, state.list.params is used for search. The change fixes the search in operations page.